backend/src/main/java/com/google/models/Alert.java

```diff
@@ -91,11 +93,14 @@ public Entity toEntity() {
   public static Alert createAlertFromEntity(Entity alertEntity) {
     List<EmbeddedEntity> listEE = (List<EmbeddedEntity>) alertEntity.getProperty(ANOMALIES_LIST_PROPERTY);
 
-    List<Anomaly> listAnomaly = new ArrayList<Anomaly>();
-    if (listEE != null) {
-      listEE.forEach(embeddedAnomaly -> listAnomaly.add(Anomaly.createAnomalyFromEmbeddedEntity(embeddedAnomaly)));
+    if (listEE == null) {
+      throw new AssertionError("No list of anomaly embedded entity found.");
     }
 
+    List<Anomaly> listAnomaly = listEE.stream()
+        .map(embeddedAnomaly -> Anomaly.createAnomalyFromEmbeddedEntity(embeddedAnomaly))
```

Comment:
I think you can do map(Anomaly::createAnomalyFromEmbeddedEntity) here too

Reply:
yep, this works too! thanks!